src/libraries/System.Net.Connections/Directory.Build.props

```diff
@@ -2,6 +2,5 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
-    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
These need to be specified here now: https://github.com/dotnet/runtime/blob/master/src/libraries/NetCoreAppLibrary.props#L4

Reply:
Thanks!